framework/transport/ieee1905_transport/ieee1905_transport_broker.cpp

```diff
@@ -240,7 +238,7 @@ bool BrokerServer::handle_subscribe(std::shared_ptr<Socket> &sd,
         }
 
         // Add to the list of requested types
-        types_stream << "0x" << std::hex << msg_type.value << std::string(" ");
+        log_types << "0x" << std::hex << msg_type.value << std::string(" ");
```

Comment:
```suggestion
        log_types << "0x" << std::hex << msg_type.value << " ";
```
why isn't this enough?
(nitpick)

Reply:
Should be enough